src/openrct2-ui/windows/NetworkStatus.cpp

```diff
@@ -25,9 +25,9 @@ enum WINDOW_NETWORK_STATUS_WIDGET_IDX {
 };
 
 static rct_widget window_network_status_widgets[] = {
-    { WWT_FRAME,            0,  0,      440,    0,      90,     0xFFFFFFFF,                 STR_NONE },                                 // panel / background
-    { WWT_CAPTION,          0,  1,      438,    1,      14,     STR_NONE,                   STR_WINDOW_TITLE_TIP },                     // title bar
-    { WWT_CLOSEBOX,         0,  427,    437,    2,      13,     STR_CLOSE_X,                STR_CLOSE_WINDOW_TIP },                     // close x button
+    MakeWidget({  0, 0}, {441, 91}, WWT_FRAME,    0                                   ), // panel / background
```

Comment:
This looks like the window shim. If so, it should be replaced with a call to WINDOW_SHIM,

Reply:
Yes, I noticed there are quite a few widget arrays that do not use window shims yet. I would prefer to replace this, along with the 'old' `WINDOW_SHIM` macros, in another PR, though. Let's stay focused.